packages/website-eslint/package.json

```diff
@@ -31,9 +31,7 @@
     "@typescript-eslint/eslint-plugin": "6.4.0",
     "@typescript-eslint/parser": "6.4.0",
     "@typescript-eslint/scope-manager": "6.4.0",
-    "@typescript-eslint/types": "6.0.0",
     "@typescript-eslint/typescript-estree": "6.4.0",
-    "@typescript-eslint/utils": "6.0.0",
```

Comment:
[Question] Merge issue?

Reply:
no, its a bug fix `@typescript-eslint/utils` is both in dependencies and dev dependencies, additionally this one has wrong version